fp.pretty/Interface_UART_RS485_Renesas_SMD.pretty/TSOT-23-6_B2.8x2.9X1.45mm_P0.95mm.kicad_mod

```diff
@@ -28,8 +28,8 @@
   (pad 6 smd rect (at -0.95 -1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
   (pad 5 smd rect (at 0 -1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
   (pad 1 smd rect (at -0.95 1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
-  (model "C:/Users/anique alamgir/Downloads/User Library-SOT23-6-1.step"
-    (offset (xyz -1 -1 0))
+  (model ${KISYS3DMOD}/kicad-packages3D/blob/master/Package_TO_SOT_SMD.3dshapes/SOT-23-6
```

Comment:
Just need to adjust this path to:
* remove the `kicad-packages3D/blob/master/` part
* point to the `.wrl`

Reply:
also please move this file to:

fp.pretty/TSOT-23-6_B2.8x2.9X1.45mm_P0.95mm.kicad_mod